selfdrive/selfdrived/selfdrived.py

```diff
@@ -41,8 +43,15 @@
 IGNORED_SAFETY_MODES = (SafetyModel.silent, SafetyModel.noOutput)
 
 
-def check_lateral_iso_violation(car_control, controls_state, car_state, live_parameters):
-  roll_compensated_lateral_accel = controls_state.curvature * car_state.vEgo ** 2 - live_parameters.roll * ACCELERATION_DUE_TO_GRAVITY
+def check_lateral_iso_violation(car_control, controls_state, car_state, live_parameters, live_calibration, live_pose, calibrator):
+  calibrator.feed_live_calib(live_calibration)
+  device_pose = Pose.from_live_pose(live_pose)
+  calibrated_pose = calibrator.build_calibrated_pose(device_pose)
```

Comment:
@fredyshox do you know why we stopped publishing the calibrated values?

Reply:
you don't want redundant info in the logs

Reply:
Sure, but you won't be able to debug this with PJ at the moment.